### PR TITLE
Project scope and architectural change process

### PR DESCRIPTION
**Pull request by seroyer**
This is in Draft for intentionally to prevent this from merging too soon.

Signed-off-by: Steven Royer <sroyer@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md
Comment (mestery) [inline CONTRIBUTING.md]: Approved by two maintainers, not reviewers. Reviewers can review, but maintainers need to approve for merge.

**Review by mestery — APPROVED** [review]
I am fine by this, though it does slow things down. But we can try it for a while and if it becomes an impediment to progress, we should revisit. Thanks Steve!

**Comment by sandersms** [comment]
For the Architecture level items this looks good.  Should we add that an Issue will be opened for the architecture change and marked with the Architecture label?  This way we can still have the general items that are contributed be reviewed and merged without the time limit delay with the approval.

File: CONTRIBUTING.md
Comment (shachartal) [inline CONTRIBUTING.md]: We have somewhat non-overlapping working days and time zones (for instance, an email to opi-dev about this pull request came into my inbox after 9pm on my last workday of the week, and I only noticed it after a long work day the following week).

I understand the desire to be able to move faster, but let's make this time window somewhat longer for architecture changes. These are material changes, they will not be made often, and they can be left to bake slightly longer. I propose making this 4 calendar days.

Comment (glimchb) [reply CONTRIBUTING.md]: 4 days sounds reasonable

Comment (seroyer) [reply CONTRIBUTING.md]: I'm also ok with 4 days.

Comment (mestery) [reply CONTRIBUTING.md]: 4 days is fine as well. As with anything, this can all be changed based on how well or not it's working.

Comment (sandersms) [reply CONTRIBUTING.md]: 4 days is fine - should we clarify if this is workdays vs calendar days?

Comment (timmichels) [reply CONTRIBUTING.md]: I agree Mark - work days. If that is the consensus, the doc should be updated before the vote on Thursday.

Comment (seroyer) [reply CONTRIBUTING.md]: Saying "work days" is tricky because different regions have different work days.  So I think I'll say 4 calendar days but I think the maintainers will be responsible for being understanding of extenuating circumstances like major holidays etc...

**Review by mestery — APPROVED** [review]
image

**Review by glimchb — APPROVED** [review]
looks good after this iteration, approved

**Review by timmichels — APPROVED** [review]
Looks good

**Pull request merged**
